packages/devtools_app/pubspec.yaml

```diff
@@ -130,3 +129,5 @@ dependency_overrides:
     path: ../devtools_shared #OVERRIDE_FOR_DEVELOPMENT
   devtools_testing: #OVERRIDE_FOR_DEVELOPMENT
     path: ../devtools_testing #OVERRIDE_FOR_DEVELOPMENT
+  devtools:  #OVERRIDE_FOR_DEVELOPMENT
+    path: ../devtools  #OVERRIDE_FOR_DEVELOPMENT
```

Comment:
why is this necessary if the dev dependency on devtools already depends on devtools from path? is devtools a transitive dep somewhere?

Reply:
I've now tweaked the other dependency to not be a path dependency for clarity. Using dependency overrides makes the version solving a little less frustrating.